sample-app/redisearch-docker/dataset/import-data.sh

```diff
@@ -0,0 +1,6 @@
+sleep 15
+redis-cli -h redisearch -p 6379 < /dataset/import_actors.redis
+redis-cli -h redisearch -p 6379 < /dataset/import_movies.redis
+redis-cli -h redisearch -p 6379 < /dataset/import_users.redis
+redis-cli -h redisearch -p 6379 < /dataset/import_create_index.redis
+exit
```

Comment:
This is free, no need to actively call exit.

Reply:
removed the exit